src/tools/genpolicy/src/config_map.rs

```diff
@@ -76,6 +76,27 @@ pub fn get_value(value_from: &pod::EnvVarSource, config_maps: &Vec<ConfigMap>) -
     None
 }
 
+//eg ["key1=value1", "key2=value2"]
+pub fn get_values(config_map_name: &str, config_maps: &Vec<ConfigMap>) -> Option<Vec<String>> {
+    for config_map in config_maps {
+        if config_map_name == &config_map.metadata.name.clone().unwrap() {
```

Comment:
I would try to move this code into a ConfigMap member function similar to get_value() and use "if let Some(my_name) = &self.metadata.name {" instead of this cloning.

Reply:
Added [get_key_value_pairs](https://github.com/microsoft/kata-containers/pull/128/files#diff-316862490937e09fc44d29ee85ceca9a8dfeda9b1b299a4e8c4c05ba12b3b470R68) as a member function that may get used elsewhere. I may be wrong here but I feel get_values still needs to be outside ConfigMap implementation, because we are [trying to find the right configmap out of a collection](https://github.com/microsoft/kata-containers/pull/128/files#diff-742867f26fd08a8659faf8e8d6a4819cba63db91a7f7b8848f65cd56e1a2fc32R648). Similar to this [get_value](https://github.com/microsoft/kata-containers/pull/128/files#diff-316862490937e09fc44d29ee85ceca9a8dfeda9b1b299a4e8c4c05ba12b3b470R82)